R/Accelerated_Failure_time_model.qmd

```diff
@@ -0,0 +1,133 @@
+df <- data.frame(time = lung$time, age = lung$age, sex = lung$sex, ph.ecog = lung$ph.ecog, status=lung$status)
+df$surv_times <- predict(aft_model, newdata = df)
+
+# Plot the survival curves based on the AFT model
+ggsurvplot(survfit(Surv(surv_times, status) ~ 1, data = df),
+           data = df, xlab = "Time", ylab = "Survival Probability")
+```
+
+The survival curve plotted based on the AFT model for the lung dataset provides valuable insights into the estimated survival probabilities over time. The curve illustrates how the probability of survival changes as time progresses.
+
+## Example of AFT model using "Weibull Distribution"
+
+```{r}
+attach(ovarian)
+
+# Fit an AFT model using lognormal distribution
+model_aft_wb <- survreg(Surv(futime, fustat) ~ age + resid.ds + rx, data = ovarian, dist = "weibull")
+# Model summary
+summary(model_aft_wb)
+```
+
+## Acceleration Factor Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratio <- exp(coef(model_aft_wb))
+hazard_ratio
+```
+
+## Interpretation
+
+-   For age, a hazard ratio of 0.93 indicates that for each one-unit increase in age, the survival time is decelerated by a factor of 0.93(or a 7% decrease)
+
+-   For residual disease status, a hazard ratio of 0.60 suggests that a decrease in residual disease status is associated with a 40% lower hazard rate or decelerated survival time.
+
+-   a hazard ratio of 1.67 suggests a 67% higher hazard rate or accelerated survival time for patients receiving a different type of radiation therapy (rx = 2) compared to the reference group (rx = 1).
+
+## Survival Curve Plotting on 'Ovarian' Dataset
+
+```{r}
+
+# Fit the AFT model (weibull distribution) on your data
+model_aft <- survreg(Surv(futime, fustat) ~ age + resid.ds + rx, data = ovarian, dist = "weibull")
+
+# Create survival curves for different levels of predictor variables
+plot_data <- with(ovarian, data.frame(age = seq(min(age), max(age), length.out = 100),
+                                      resid.ds = mean(resid.ds),
+                                      rx = mean(rx)))
+
+# Predict survival times based on the AFT model
+plot_data$survival <- predict(model_aft, newdata = plot_data)
+
+# Plot the survival curves
+ggplot(plot_data, aes(x = age, y = survival, color = factor(rx), linetype = factor(rx))) +
+  geom_line() +
+  labs(x = "Age", y = "Survival Probability", color = "Radiation Therapy", linetype = "Radiation Therapy") +
+  scale_linetype_manual(values = c("solid", "dashed", "dotted")) +
+  scale_color_manual(values = c("blue", "red", "green"))
+```
+
+The survival curve plotted based on the AFT model for the ovarian dataset how the probability of survival changes as age increases.
+
+## Conclusion
+
+In AFT models unlike Cox Ph model ,survival times follow assumed parametric distribution(e.g. extreme value distribution for Weibull, logistic distribution for log-logistic, standard normal for log-normal).
```

Comment:
In AFT models, unlike Cox proportional hazards models, survival times follow an assumed parametric distribution (e.g., extreme value distribution for Weibull, logistic distribution for log-logistic, standard normal for log-normal), directly modelling the effect of covariates on the time scale.